run_dir/design/running_notes_tab.html

```diff
@@ -6,11 +6,25 @@
                 <div class="panel-heading">Add New Running Note</div>
                 <div class="panel-body">
                     <div class="row">
-                        <div class="col-md-6">
+                        <div class="col-md-7 form-inline">
+                          <label>Choose category:</label>
+                          <div class="btn-toolbar" data-toggle="buttons" style="margin-top:5px">
+                             <button class="btn btn-xs btn-info btnCatFilter" value="Decision" data-toggle="tooltip" title="Use this category when an executive decision has been made">Decision &nbsp;<span class="fa fa-thumbs-up"</span></button>
+                             <button class="btn btn-xs btn-success btnCatFilter" value="Lab" data-toggle="tooltip" title="Use this category for lab-related work">Lab &nbsp;<span class="fa fa-bong"</span></button>
+                             <button class="btn btn-xs btn-warning btnCatFilter" value="Bioinformatics" data-toggle="tooltip" title="Use this category for all bioinformatics work">Bioinformatics &nbsp;<span class="fa fa-laptop-code"</span></button>
+                             <button class="btn btn-xs btn-usr btnCatFilter" value="User Communication" data-toggle="tooltip" title="Use this category for notes influenced by user-contact">User Communication &nbsp;<span class="fa fa-people-arrows"</span></button>
+                             <button class="btn btn-xs btn-danger btnCatFilter" value="Administration" data-toggle="tooltip" title="Use this category for notes involving documentation">Administration &nbsp;<span class="fa fa-folder-open"</span></button>
+                             <button class="btn btn-xs btn-imp btnCatFilter" value="Important" data-toggle="tooltip" title="Use this category when a note needs to be highlighted">Important &nbsp;<span class="fa fa-exclamation-circle"</span></button>
+                             <button class="btn btn-xs btn-devi btnCatFilter" value="Deviation" data-toggle="tooltip" title="Use this category for notes about a deviation">Deviation &nbsp;<span class="fa fa-frown"</span></button>
+                          </div>
```

Comment:
* Instead of giving every element a `.btnCatFilter` class, just give the one parent element a class and refer to each child (here: `.rn-cat-filter button`).
* No need to specify `value` when it's the same as the `.text()` contents of each button
* Spacing using `&nbsp;` is a nasty hack. With the newer bootstrap you can add margins with helper classes like `.ml-2` on the icon (`ml-2` = margin left x 2). Here we can just add some custom CSS to the stylesheet.

```suggestion
                          <div class="btn-toolbar rn-cat-filter" data-toggle="buttons" style="margin-top:5px">
                             <button class="btn btn-xs btn-info" data-toggle="tooltip" title="Use this category when an executive decision has been made">Decision <span class="fa fa-thumbs-up"</span></button>
                             <button class="btn btn-xs btn-success" data-toggle="tooltip" title="Use this category for lab-related work">Lab <span class="fa fa-bong"</span></button>
                             <button class="btn btn-xs btn-warning" data-toggle="tooltip" title="Use this category for all bioinformatics work">Bioinformatics <span class="fa fa-laptop-code"</span></button>
                             <button class="btn btn-xs btn-usr" data-toggle="tooltip" title="Use this category for notes influenced by user-contact">User Communication <span class="fa fa-people-arrows"</span></button>
                             <button class="btn btn-xs btn-danger" data-toggle="tooltip" title="Use this category for notes involving documentation">Administration <span class="fa fa-folder-open"</span></button>
                             <button class="btn btn-xs btn-imp" data-toggle="tooltip" title="Use this category when a note needs to be highlighted">Important <span class="fa fa-exclamation-circle"</span></button>
                             <button class="btn btn-xs btn-devi" data-toggle="tooltip" title="Use this category for notes about a deviation">Deviation <span class="fa fa-frown"</span></button>
                          </div>
```